apps/mark-scan/frontend/src/pages/pat_device_identification/pat_device_identification_page.tsx

```diff
@@ -94,7 +94,7 @@ export function PatDeviceIdentificationPage({
             {statusStrings[currentStepId]}
           </P>
         </DiagnosticScreenHeader>
-        <StepContainer>{steps[currentStepId]}</StepContainer>
+        <PatStepContainer>{steps[currentStepId]}</PatStepContainer>
```

Comment:
The PAT "diagnostic" here is a calibration flow for the voter. It still needs to be wired up to the `diagnostic_record` construct for system admins.